### DISCUSSION TITLE
Add (automated) RBF and CPFP functionality in Bisq Wallet, to also lower mining fees

### DISCUSSION THREAD

**Discussion by w0000000t** [bisq-network/bisq#5890] [Ideas]
This could become a way to solve, maybe, the high fees problem that on one side are needed to insure trades work in a fluent fashion, but on the other means usually overspending especially during these low mempool times.

RBF could help where you take (also make?) an offer using a small fee, and if the deposit hangs because a flood of new transactions happen to get in line before you (and Bisq detects it), you can re-issue the security deposit transaction with a higher fee.

"Manual" RBF and CPFP could be useful when respectively withdrawing funds from wallet (allowing for 1sat/vB withdraw fee) and receving deposits from externall wallets (even if you could also use RBF from external wallet in this case, but the more options the better)

**Comment by MwithM** [comment]
RBF is not available for BitcoinJ.

https://github.com/bisq-network/proposals/issues/343#issuecomment-954710781
